journal/updated/09-chapter-8.adoc

```diff
@@ -274,13 +278,14 @@ and never moved out of the pure order of the Gospel.
 I left my home,
 having for some time felt an inclination to visit a few meetings in Warwickshire,
 and Ann Byrd coming our way and being very desirous of my company,
-and she having a concern to visit the said meetings, we went together in true unity,
-and were indeed fellow-labourers in the Gospel though very deep was our travail,
+having a concern to visit the said meetings, we went together in true unity,
+and were indeed fellow-labourers in the Gospel, though very deep was our travail,
 and great the exercise of our minds,
 because of the lukewarmness and indifferency which
-prevail among a people who have been so favoured;
-yea, a people beloved of God and chosen of him before all the families of the earth,
-and if there be not a reformation, them will he punish.
+prevail among a people who have been so favoured.
+Yes, these were a people beloved of God and chosen
+of Him before all the families of the earth;
+and if there be not a reformation, them will He punish.
```

Comment:
I would probably change this line because it sounds weird. Maybe something like: so that surely, if there is not a reformation, they will be punished by Him.